Development environment setup/Local Emails.md

```diff
@@ -0,0 +1,9 @@
+To avoid spamming or sending emails from your local enviroment to the real world users you should always setup your local enviroment to use a local mail server. We recommend using [Mailpit](https://mailpit.axllent.org/) or [Mailhog](https://github.com/mailhog/MailHog).
```

Comment:
Mailhog is no longer maintained - at least not since 2020. Maybe we should just mention Mailpit here.